bin/codecov

```diff
@@ -115,22 +115,16 @@ var argv = require('yargs') // eslint-disable-line
 
 const start = Date.now()
 
-app.log(`Start of uploader: ${start}...`, { level: 'debug', argv })
+app.verbose(`Start of uploader: ${start}...`, argv.verbose)
 app
   .main(argv)
   .then(() => {
     const end = Date.now()
-    app.log(`End of uploader: ${end - start} milliseconds`, {
-      level: 'debug',
-      argv,
-    })
+    app.verbose(`End of uploader: ${end - start} milliseconds`, argv.verbose)
   })
   .catch(error => {
-    app.log(`Error!: ${error}`, { level: 'error', argv })
+    app.error(`Error!: ${error}`)
```

Comment:
Is `error` exported in ../src/index.js?

Reply:
@*#. No.

Reply:
> > > "checkJs": true
> > 
> > 
> > Ah, missed that one, thanks :)
> 
> Yup. A recent change (TypeScript 4.3 or 4.4, I forget) made `checkJs` imply `allowJs` so it’s now the one to use.

I read that, and mentally reversed them, which is why I only had the one